docs/README.md

```diff
@@ -1,2 +1,294 @@
+5. Is there any [upgrade process](https://github.com/johnwhitton/horizon/tree/main/scripts/upgrade) and if so when is it needed?
+6. What is the purpose of `node index.js Bridge deployFakeClient`
+7. What is the purpose of `node index.js Bridge change`
+8. Are there any dependencies on tools or scripts for deployment?
+9. What is the purpose of ``yarn libInit` in the [elc installation instructions](https://github.com/johnwhitton/horizon/tree/main/tools/elc#installation-instructions)
+
+## Deploying the Relayer
+The eht2hmy-relayEth2Hmy relay downloads the Ethereum block headers, extract information and relay it to ELC smart contract on Harmony. It exists in the [tools folder](https://github.com/johnwhitton/horizon/tree/main/tools/eth2hmy-relay) and can be run via the [cli](https://github.com/johnwhitton/horizon/tree/main/cli#ethereum-block-relay-cli)
+
+**Original documentation**
+Original documentation can be found [here](https://github.com/johnwhitton/horizon/tree/main/cli#ethereum-block-relay-cli) and says
+
+> Before using the CLI, fill in the private key into `.env` and execute `source .env`.
+1. `node index.js ethRelay getBlockHeader` get block header from ethereum.
+```
+node index.js ethRelay getBlockHeader https://ropsten.infura.io/v3/<project-id> 11266872
+```
+2. `node index.js ethRelay relay` constantly relay blocks from Ethereum to Harmony.
+
+**Questions**
+
+1. Can we simply run `node index.js ethRelay relay` to run the relayer?
+2. Are there any other components which need to be run?
+3. Are there any other infrastrcture tasks which need to be setup prior to running the relayer?
+
+
+## End to End Testing
+End to End testing currently consists of mapping ERC20 tokens from Harmony to Ethereum and transferring them accross the chains. This is done via the [ClI](https://github.com/johnwhitton/horizon/tree/main/cli) using the following commands
+
+5. `node index.js Bridge map` map ERC20 from ethereum to harmony.
+6. `node index.js Bridge crossTo` cross transfer ERC20 from ethereum to harmony.
+7. `node index.js Bridge crossBack` cross transfer HRC20 from harmony back to ethereum.
+
+There are also two scripts written to facilitate testing. 
+1. [test.js](https://github.com/johnwhitton/horizon/blob/main/scripts/test.js)
+2. [newtests.js](https://github.com/johnwhitton/horizon/blob/main/scripts/newtest.js)
+
+Also [end2end.js](https://github.com/johnwhitton/horizon/blob/main/scripts/end2end.js) which does not have any code in it.
+
+**Original Documentation**
+Original documentation for the CLI contains [Bridge CLI](https://github.com/johnwhitton/horizon/tree/main/cli#bridge-cli), [Ethereum Receipt Prove CLI](https://github.com/johnwhitton/horizon/tree/main/cli#ethereum-receipt-prove-cli) and [Ethereum Receipt Verifier CLI](https://github.com/johnwhitton/horizon/tree/main/cli#ethereum-receipt-verifier-cli).
+
+The assumption is that you need to just run the map, crossTo and crossBack functions.
+
+**Questions**
```

Comment:
Move elsewhere? As to the questions, I suppose Bridge subcommands (the three you mentioned) can be used to do human-driven testing once deployment is complete. But they don't seem to be good for any automated testing. I don't think there is any end-to-end testing elsewhere